### PR TITLE
[CBRD-24609] When loading objects files online, if the schema name is not specified, the schema name of the current session must be used.

### PR DESCRIPTION
**Pull request by youngjinj**
http://jira.cubrid.org/browse/CBRD-24609

When loading objects files online, if the schema name is not specified, the schema name of the current session must be used.

### CODE REVIEW

File: src/loaddb/load_server_loader.cpp [hunk 1]
```diff
@@ -117,8 +117,28 @@ namespace cubload
     cubthread::entry &thread_ref = cubthread::get_entry ();
     LC_FIND_CLASSNAME found = LC_CLASSNAME_EXIST;
     LC_FIND_CLASSNAME found_again = LC_CLASSNAME_EXIST;
+    const char *dot = strchr (class_name, '.');
```
Comment (shparkcubrid) [inline src/loaddb/load_server_loader.cpp]: Is there any other way to check? For example checking the flag

**Review by hgryoo — APPROVED** [review]

**Review by beyondykk9 — APPROVED** [review]

**Review by hyunikn — APPROVED** [review]

File: src/loaddb/load_server_loader.cpp [hunk 2]
```diff
@@ -117,8 +117,28 @@ namespace cubload
...
+
+    if (dot != NULL)
```
Comment (ctshim) [inline src/loaddb/load_server_loader.cpp]:
```suggestion
    if (strchr (class_name, '.'))
```

**Review by ctshim — APPROVED** [review]

**Review by shparkcubrid — APPROVED** [review]

File: src/loaddb/load_server_loader.cpp (hunk 1 above)
Comment (youngjinj) [reply src/loaddb/load_server_loader.cpp]: It doesn't seem to be a problem with loaddb only.
I will think more through other issues.

File: src/loaddb/load_server_loader.cpp (hunk 2 above)
Comment (youngjinj) [reply src/loaddb/load_server_loader.cpp]: Thank you. I will change it.

**Pull request merged**